cura/CuraApplication.py

```diff
@@ -725,8 +695,12 @@ def run(self):
 
         self.exec_()
 
+    def __setUpSingleInstanceServer(self):
+        if self._use_single_instance:
```

Comment:
Seems like a bit of an overkill to put startServer in a function and check _use_single_single two times (it is also checked before calling this function)